server/sonar-web/src/main/js/apps/quality-profiles/details/ChangeProjectsForm.js

```diff
@@ -64,10 +72,32 @@ export default ModalFormView.extend({
         deselect: translate('quality_profiles.projects.deselect_hint')
       }
     });
-  },
+  }
+
+  render() {
+    const header = translate('projects');
+
+    return (
+      <Modal
+        isOpen={true}
+        contentLabel={header}
+        className="modal"
+        overlayClassName="modal-overlay"
+        onRequestClose={this.props.onClose}>
+
+        <div className="modal-head">
+          <h2>{header}</h2>
+        </div>
+
+        <div className="modal-body">
+          <div id="profile-projects" ref={node => (this.container = node)} />
+        </div>
+
+        <div className="modal-foot">
+          <a href="#" onClick={this.handleCloseClick}>{translate('close')}</a>
```

Comment:
Yes, because all other modals have two options: submit or cancel. Here there is no explicit submit action, when you select/deselect a project, it is submitted automatically.